src/LodeRunner.UI/src/components/App/index.js

```diff
@@ -28,6 +30,79 @@ function App() {
       });
   }, [fetchClientsCount]);
 
+  useEffect(() => {
+    // fetch(`${process.env.REACT_APP_SERVER}/api/configs`)
```

Comment:
I'm thinking we should put the retrieval code in a "service" or "data access" entity.  Then this would be able to import the entity and use it and the details of the retrieval being either a remote call or a read from a local file are hidden there, but the code in index.js (or other files) would remain the same.